gravitee-am-test/package-lock.json

```diff
@@ -4219,9 +4218,9 @@
       }
     },
     "node_modules/entities": {
-      "version": "4.3.1",
-      "resolved": "https://registry.npmjs.org/entities/-/entities-4.3.1.tgz",
-      "integrity": "sha512-o4q/dYJlmyjP2zfnaWDUC6A3BQFmVTX+tZPezK7k0GLSU9QYCauscf5Y+qcEPzKL+EixVouYDgLQK5H9GrLpkg==",
+      "version": "4.4.0",
+      "resolved": "https://registry.npmjs.org/entities/-/entities-4.4.0.tgz",
+      "integrity": "sha512-oYp7156SP8LkeGD0GF85ad1X9Ai79WtRsZ2gxJqtBuzH+98YUV6jkHEKlZkMbcrjJjIVJNIDP/3WL9wQkoPbWA==",
```

Comment:
Don't commit this file.

Reply:
this point is not clear to me, will discuss with you.

Reply:
Did you intend to commit the package-lock.json? It gets updated when you run `npm install` but we generally don't commit those changes unless there was a reason to. If it needs updating to run your test, that's fine.